README.md

```diff
@@ -464,6 +464,81 @@ repository, and `ZULIP_GIT_REF` to be any ref name in that repository
 
 Then stop and restart the container as described in the previous section.
 
+### Upgrading zulip/zulip-postgresql to 14
+
+These instructions assume that you have not changed the default
+Postgres data path (`/opt/docker/zulip/postgresql/data`) in your
+`docker-compose.yml`.  If you have changed it, please replace all
+occurences of `/opt/docker/zulip/postgresql/data` with your path.
+
+1. Make a backup of your Zulip Postgres data dir.
+
+2. Stop all Zulip containers, except the postgres one (e.g. use
+   `docker stop` and not `docker-compose stop`).
```

Comment:
Can we given the precise command to run?

Reply:
docker stop can take a list of IDs but those aren't static.

Reply:
OK; but maybe we can give the command for listing the contains and then say to stop all the ones that aren't the postgres one?

Reply:
It looks like we're still stopping `memcached rabbitmq redis` -- those don't talk to PostgreSQL.  Any reason to not only `docker-compose stop zulip`?

Reply:
pushed this change. Untested.